pysatCDF/_cdf.py

```diff
@@ -569,7 +569,7 @@ def _process_return_multi_z_attr(self, data, attr_names, var_names,
                             chars.append('*')
                     self.meta[var_name][attr_name] = ''.join(chars).rstrip()
                 else:
-                    self.meta[var_name][attr_name] = data[i, 0:num_e]
```

Comment:
I'm happy with this change. But I don't think I understand why this was producing an error.
[The documentation for indexing](https://numpy.org/doc/stable/user/basics.indexing.html) has examples in the "Advanced Indexing" section that look the same as `data[i, 0:num_e]`.

Reply:
I also see data access examples like data[I, j] in the documentation. I don't quite understand why this change is needed.